stripe/src/main/java/com/stripe/android/paymentsheet/PaymentOptionsActivity.kt

```diff
@@ -227,7 +228,6 @@ internal class PaymentOptionsActivity : BaseSheetActivity<PaymentOptionResult>()
         supportFragmentManager.executePendingTransactions()
```

Comment:
On line 221 I don't think you need the executePendingTransactions any more.   Same thing will be true in the PaymetSheetActivity.  There is also a test associated with this (once you remove it).

To verify, use the custom sheet with a customer, add an unsaved card, and click select (it should jump right to the populated unsaved card).  Do this 10+ times and make sure the close/back button in the toolbar looks correct.

I like the watch the backstack change.

Reply:
Right, if this is really just to get an accurate backstack count then we don't need it. It seems to work correctly without it.